src/classes/ResponseWrapper.cls

```diff
@@ -0,0 +1,23 @@
+/**
+ * Created by User on 12.10.2017.
+ */
+
+public with sharing class ResponseWrapper {
```

Comment:
not very informative name of wrapper

Reply:
it is better to use wrapper postfix for standard objects, custom wrapper should not contain this postfix